sqldelight-gradle-plugin/src/main/kotlin/com/squareup/sqldelight/gradle/SqlDelightTask.kt

```diff
@@ -70,7 +70,9 @@ open class SqlDelightTask : SourceTask() {
       inputFileDetails.file.parseThen { parsed ->
         var status: Status = sqldelightValidator.validate(parsed, symbolTable)
         if (status is Status.ValidationStatus.Invalid) {
-          errors.addAll(status.errors.map {
+          errors.addAll(status.errors
+              .distinctBy { it.originatingElement.start.startIndex to it.originatingElement.stop.stopIndex }
```

Comment:
Can we upstream this logic into the validator so that it's not repeated in both plugins?